lib/exec.ts

```diff
@@ -241,7 +241,9 @@ export function getNsJailOptions(
 
     const env: Record<string, string> = {...options.env, HOME: homeDir};
     if (options.ldPath) {
-        jailingOptions.push(`--env=LD_LIBRARY_PATH=${options.ldPath.join(path.delimiter)}`);
+        const transform = filenameTransform || (x => x);
```

Comment:
We should probably not copy this in two places. Let's pull this out to top scope and use it in both places.